src/cycax/cycad/features.py

```diff
@@ -141,13 +141,20 @@ def swap_yz(self, rot: float, rotmax: list) -> list:
 nut_specifications: dict[
     str, dict[str, float]
 ] = {  # This is a global variable that will be used to cut the nuts by the OpenSCAD engine.
-    "M3": {
+    "MISO3": {
```

Comment:
This is warped to much. "M3" is the name, and is lost in this scheme.  Put ISO before or after M3 not in the middles. MISO reminds me to much of antenna design.